src/recon_test/test_consistency_with_root.cxx

```diff
@@ -207,7 +207,7 @@ setup()
 
   // Failure conditioner and recording
   nonTOF_distance_threshold = 1.5 * norm(grid_spacing); // Using norm(grid_spacing) as a nonTOF_distance_threshold
-  TOF_distance_threshold = 2.5 * norm(grid_spacing); // Using norm(grid_spacing) as a tof_distance_threshold
+  TOF_distance_threshold = 3.3 * norm(grid_spacing); // Using norm(grid_spacing) as a tof_distance_threshold
```

Comment:
Can you update the comment here? Why `3.3`?

Reply:
After having changed source positions in GATE and seeing the test-results didn't change dramatically, we agreed that I should use the lowest number for which the test passes. That's what I put in the comments now. I hope this is sufficient, please let me know if I should add more info. Thanks!